rct229/rulesets/ashrae9012019/section19/section19rule13.py

```diff
@@ -0,0 +1,255 @@
+                                == LIGHTING_SPACE.LABORATORY_EXCEPT_IN_OR_AS_A_CLASSROOM
+                            )
+                            else False
+                        )
+
+                    # calculate proposed_supply_airflow
+                    for terminal_p in find_all(
+                        '$.buildings[*].building_segments[*].zones[?(@.id == "{zone_id_b}")].terminals[*]',
+                        rmi_p,
+                    ):
+                        zone_info[zone_id_b]["proposed_supply_airflow"] += getattr_(
+                            terminal_p, "Terminal", "primary_airflow"
+                        )
+
+                for terminal_b in dict_of_zones_and_terminal_units_served_by_hvac_sys_b[
+                    hvac_id_b
+                ]["terminal_unit_list"]:
+                    design_heating_supply_air_temp_setpoint = getattr_(
+                        terminal_b,
+                        "Terminal",
+                        "supply_design_heating_setpoint_temperature",
+                    )
+                    design_cooling_supply_air_temp_setpoint = getattr_(
+                        terminal_b,
+                        "Terminal",
+                        "supply_design_cooling_setpoint_temperature",
+                    )
+
+                    if zone_has_lab_space:
+                        zone_info[zone_id_b][
+                            "all_design_setpoints_delta_Ts_are_per_reqs"
+                        ] = (
+                            True
+                            if (
+                                design_heating_supply_air_temp_setpoint
+                                - design_cooling_supply_air_temp_setpoint
+                            )
+                            != TEMP_DELTA_17F
+                            else False
+                        )
+
+                    else:
+                        zone_info[zone_id_b][
+                            "all_design_setpoints_delta_Ts_are_per_reqs"
+                        ] = (
+                            True
+                            if (
+                                design_heating_supply_air_temp_setpoint
+                                - design_cooling_supply_air_temp_setpoint
+                            )
+                            != TEMP_DELTA_20F
+                            else False
+                        )
+
+        return {
+            "zone_info": zone_info,
+        }
+
+    def is_applicable(self, context, data=None):
+        rmi_b = context.baseline
+        baseline_system_types_dict = get_baseline_system_types(rmi_b)
+
+        return any(
+            [
+                baseline_system_type_compare(system_type, applicable_sys_type, False)
+                for system_type in baseline_system_types_dict.keys()
+                for applicable_sys_type in APPLICABLE_SYS_TYPES
+            ]
+        )
+
+    class HVACRule(RuleDefinitionBase):
```

Comment:
After reading the RDS, I think the rule shall be evaluated at zone level instead of HVAC level.
The rule check is `supply_air_to_room` differences. So at room level, we shall compare the highest `heating_setpoint` to the `heating_supply_air_temp` or the lowest `cooling_setpoint` to the `cooling_supply_air_temp`

I suggest to revise the rule definition logic.